go/pkg/sysdb/metastore/db/dbcore/core.go

```
@@ -235,6 +235,10 @@ func GetDBConfigForTesting() DBConfig {
 				WithStartupTimeout(5*time.Second)),
 	)
 
+	if err != nil {
+		panic(err)
```

this is never hit, but realized any error here was swallowed during testing